src/dstack/_internal/server/schemas/runs.py

```diff
@@ -26,6 +26,7 @@ class GetRunRequest(CoreModel):
 
 class GetRunPlanRequest(CoreModel):
     run_spec: RunSpec
+    max_offers: int = Field(50, ge=1, le=1000)
```

Comment:
Should be excluded on the client if not set explicitly to handle old servers that don't support `max_offers`.